system-configuration/src/dynamic_store.rs

```diff
@@ -82,6 +82,7 @@ impl<T> SCDynamicStoreBuilder<T> {
     ///
     /// [`SCDynamicStore`]: struct.SCDynamicStore.html
     /// [`SCDynamicStoreCreateWithOptions`]: https://developer.apple.com/documentation/systemconfiguration/1437818-scdynamicstorecreatewithoptions?language=objc
+    #[must_use]
```

Comment:
Why make this `must_use`? None of the other builder methods here are `must_use`

Reply:
clippy complains when returning self without must use

Reply:
Ah. This is a nightly lint. Our CI setup on this crate is so terribly outdated. We should not run clippy on all rust versions. Would be great to have this crate migrated to github actions.